src/outputs/gb_sas.py

```diff
@@ -35,6 +35,9 @@ def output_gb_sas(
     paths = config[f"{NETWORK_OR_HDFS}_paths"]
     output_path = paths["output_path"]
 
+    # Filter out records that answer "no R&D"
+    df = df.copy().loc[~(df["604"] == "No")]
+
```

Comment:
can I just check, could or should we put this in output_main instead?

Reply:
Good question @AnneONS .  Is this relevant to all outputs, inc. short form and long form outputs?  If so, it could go in `form_output_prep()`.  If it should be included for SAS, TAU and intram outputs, it could go in `outputs_main()`.  @zorge69 please could you confirm?

Reply:
@AnneONS That change has been made